Azure-Samples/ARGUS
# Visualization for evaluation notebook

## Purpose
Adding visualization for evaluation notebook

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
See instructions in notebooks/README.md

* Test the code
This PR only changes the jupiter notebook. To test it, need to run it. It shoudl produce the following visuals

<img width="597" alt="image" src="https://github.com/user-attachments/assets/ed191bb5-e7a8-46b3-8218-2f1d5d54bfb2">
<img width="571" alt="image" src="https://github.com/user-attachments/assets/5bc19d97-f71c-456f-9920-bb1f7ca5d428">
